pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -39,37 +54,39 @@ let
 
           This package also includes the Arimo, Cousine, and Tinos fonts.
         '';
-        license = licenses.asl20;
+        license = licenses.ofl;
         platforms = platforms.all;
-        maintainers = with maintainers; [ mathnerd314 ];
+        maintainers = with maintainers; [ mathnerd314 emily ];
       };
     };
 in
 
 {
   noto-fonts = mkNoto {
-    name = "noto-fonts";
+    pname = "noto-fonts";
     weights = "{Regular,Bold,Light,Italic,BoldItalic,LightItalic}";
-    sha256 = "0kvq5ldip2ra2njlxg9fxj46nfqzq5l3n359d3kwfbsld7hixm2d";
```

Comment:
Is there a reason you removed these hashes? There was a big cleanup a while back: https://github.com/NixOS/nixpkgs/issues/27754

Reply:
Because the hash is now shared by the `fetchFromGitHub` call in `mkNoto`, rather than using fixed-output derivations for the post-processed packages.

Reply:
My personal feeling is that fixed-output derivations are error-prone and inelegant, and avoiding rebuilding some font packages when e.g. bash updates and everything needs rebuilding anyway isn't worth the deviation from the style of normal package builds.

However, it's true that rebuilding when the C compiler changes is unnecessary. A half-way solution would be to move over to using `stdenvNoCC`/`runCommandNoCC`; what do you think of that?

Reply:
From what I remember, the main concern was Hydra running out of disk space, back when it wasn't storing everything in the cloud. I don't have strong feelings either way. It seems recent font packages are free to choose fixed outputs or not.